### PR TITLE
Avoid imports error when default_language is present but no translations

### LINKED ISSUE #2073: `ValueError: translation cannot be found` when importing XLSForm
From https://www.flowdock.com/app/kobotoolbox/support/threads/yuUp1PuC4QFvtJh1U9wyhJZRVR5:
> in the past 12hrs we've had 6-7 users have asked for help through intercom about receiving this error when trying to upload an xlsform -
image
> [ACTIVITYATTENDANCESHEET.xls.zip](https://github.com/kobotoolbox/kpi/files/2577214/ACTIVITYATTENDANCESHEET.xls.zip)
> one user said they figured it out -
image
> but I looked at another users xlsform and they did not have a translation column in their form

### PR DESCRIPTION
**Pull request by noliveleger**
fixes #2073

### CODE REVIEW

File: kpi/models/asset.py [hunk 1]
```diff
@@ -302,7 +302,20 @@ def _prioritize_translation(self, content, translation_name, is_new=False):
...
         elif (not is_new) and (translation_name not in _translations):
-            raise ValueError('translation cannot be found')
+            # if there are no translations available, don't try to prioritize,
+            # just ignore the translation `translation_name`
+            if len(_translations) == 1 and _translations[0] is None:
+                return
+            else:  # Otherwise raise an error.
+                # Remove None from translations we want to display to users
+                valid_translations = [t for t in _translations if t is not None]
+                raise ValueError("Translation `{translation_name}` could not be found in your form. "
```
Comment (jnm) [inline kpi/models/asset.py]: I think we should tell the user specifically that their default language setting is the problem. Can we move this logic to `_make_default_translation_first()`? I realize `_translations = content['translations']` or similar will have to be duplicated. Then, we could make the error message something like:
```
`{translation_name}` is specified as the default language, but only these translations are present in the form: `{translations}`.
```
I like the "Please select..." language, but let's keep the message concise at the expense of courtesy :)

File: kpi/models/asset.py [hunk 2]
```diff
@@ -302,7 +302,20 @@ def _prioritize_translation(self, content, translation_name, is_new=False):
         if is_new and (translation_name in _translations):
             raise ValueError('cannot add existing translation')
         elif (not is_new) and (translation_name not in _translations):
-            raise ValueError('translation cannot be found')
+            # if there are no translations available, don't try to prioritize,
+            # just ignore the translation `translation_name`
+            if len(_translations) == 1 and _translations[0] is None:
+                return
```
Comment (jnm) [inline kpi/models/asset.py]: :+1:

File: kpi/models/asset.py (hunk 1 above)
Comment (noliveleger) [reply kpi/models/asset.py]: Done!

File: kpi/utils/log.py [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
+
 def get_default_logger():
     return stdlib_logging.getLogger("console_logger")
 
+
 # logging does not print to stdout when RAVEN/sentry is activated.
 # To avoid to call logging.getLogger everywhere, we overload logging
 logging = get_default_logger()
 
+
```
Comment (jnm) [inline kpi/utils/log.py]: PEP-8 really requires `\n\n\n` at the end of a file? I guess I don't like the BDFL's taste in whitespace :astonished:
(just commentary, not a request for any change to the code)

Comment (noliveleger) [reply kpi/utils/log.py]: No, I made the mistake. 
There are too many. There should be only 2.

**Pull request merged**